types/address_bench_test.go

```diff
@@ -11,6 +11,18 @@ import (
 	"github.com/cosmos/cosmos-sdk/types"
 )
 
+func BenchmarkAccAddressString(b *testing.B) {
+	b.ReportAllocs()
+	pkBz := make([]byte, ed25519.PubKeySize)
+	pk := &ed25519.PubKey{Key: pkBz}
+	aa := pk.Address()
+	var str string
+	for i := 0; i < b.N; i++ {
+		str = aa.String()
+	}
+	require.NotEmpty(b, str)
```

Comment:
Shouldn't you move this up to after line 21?

Reply:
I want to benchmark `String`. It's enough if we check only once, no need to do that check in the loop every time.

Reply:
Well, it's a bit unnecessary then, isn't it? Not a big deal either way

Reply:
@robert-zaremba is on point here, just check only at the end, this is an idiomatic way of checking.